src/Storefront/DependencyInjection/Configuration.php

```diff
@@ -25,7 +25,6 @@ public function getConfigTreeBuilder(): TreeBuilder
                         ->scalarNode('config_loader_id')->defaultValue(DatabaseConfigLoader::class)->end()
                         ->scalarNode('theme_path_builder_id')->defaultValue(SeedingThemePathBuilder::class)->end()
                         ->scalarNode('available_theme_provider')->defaultValue(DatabaseAvailableThemeProvider::class)->end()
-                        ->integerNode('file_delete_delay')->defaultValue(900)->end()
```

Comment:
does this need a deprecation? :thinking:

Reply:
but we can't remove it here directly right? As that would break all instances where this config was used, but IIRC the configuration has an option to mark the config option as deprecated and then we should remove it with 6.8 here, but we can already remove all the usages -> so all other removals are fine IMHO, but here we should deprecated instead

Reply:
brought it back with deprecation notice